Updated documentation related to the association with quantities. (#668)
akeneo/pim-helpcenter: content/md/first-steps/manage-type-association.md

For example, you can link a product to its accessories. The code for this association type could be “ACCESSORIES”.

There are three different association types:
*   The 1-way association (product A is associated with product B).
*   The 2-way association (when product A is associated with product B, then product B will be automatically associated with product A).

> *   The association with quantities (product A is associated with product B for a quantity of 2 and with product C with a quantity of 3)

Author: changed!

# Create an association type

:::info
You must have the permission to create an association type from the PIM. Association type must contain at least one letter.
:::

## Create a 1-way association type

To create a 1-way association type:
1.  Click on `Create association type`

> 1.  Select the code of the association type you want to create

Reviewer: Fill in is better no ?

::: success
The association type has now been created and the edit page is displayed.  
Only the labels can be changed.
:::

## Create a 2-way association type

To create a 2-way association type:
1.  Click on `Create association type`
1.  Fill in the code of the association type you want to create
1.  Set the "2-way association" button to "Yes"
1.  Click on Save

![image](../img/Settings_Associations_2-way-association-type.png)

::: success
The association type has now been created and the edit page is displayed.  
Only the labels can be changed.
:::

:::warning
The 2-way association property cannot be updated once the association type is created.
:::

## Create an association type with quantities

To create an association type with quantities:
1.  Click on `Create association type`
1.  Select the code of the association type you want to create

> 1.  Set the "Association with quantity" switch to "Yes"

Author: Switch :)

> 1.  Set the "Association with quantity" switch to "Yes"

Author: "switch"!

::: success
The association type has now been created and the edit page is displayed.  
Only the labels can be changed.
:::

:::warning
The 2-way association property cannot be updated once the association type is created.
:::

## Create an association type with quantities

To create an association type with quantities:
1.  Click on `Create association type`
1.  Select the code of the association type you want to create

> 1.  Set the "Association with quantity" button to "Yes"

Reviewer 2: ```suggestion
1.  Set the "Association with quantity" switch to "Yes"
```

::: warning
You cannot change the code once you created the association type.    

> Neither will you be able to update the 2-way association nor the association with quantities.

Reviewer 2: ```suggestion
Neither will you be able to change it to a 2-way association nor an association with quantities.
```
